<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeedPriceDialog</class>
 <widget class="QDialog" name="FeedPriceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>556</height>
    </rect>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>138</y>
      <width>520</width>
      <height>280</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>资产名:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="myPriceLineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QToolButton" name="feedBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>120</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>喂价</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>当前喂价:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>新的喂价:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>选择喂价账户:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>100</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>账户余额:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="accountComboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>/UB</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>/UB</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentPriceLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="accountBalanceLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>100</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>UB</string>
     </property>
    </widget>
    <widget class="QComboBox" name="tokenComboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>53</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>495</x>
       <y>12</y>
       <width>13</width>
       <height>13</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
